asreview/webapp/api.py

```diff
@@ -1046,23 +1046,22 @@ def api_get_progress_history(project_id):
     return response
 
 
-@bp.route('/project/<project_id>/progress_efficiency', methods=["GET"])
-def api_get_progress_efficiency(project_id):
+@bp.route('/project/<project_id>/progress_recall', methods=["GET"])
+def api_get_progress_recall(project_id):
     """Get cumulative number of inclusions by ASReview/at random"""
 
     try:
-        statistics = get_data_statistics(project_id)
-        labeled = read_label_history(project_id)
-        data = []
-        for [key, value] in labeled:
-            data.append(value)
+
```

Comment:
This needs `project_path = get_project_path(project_id)`.

Reply:
Thanks